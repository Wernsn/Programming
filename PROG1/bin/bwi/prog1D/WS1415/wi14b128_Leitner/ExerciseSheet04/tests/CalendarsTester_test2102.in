21
1582 10 15
