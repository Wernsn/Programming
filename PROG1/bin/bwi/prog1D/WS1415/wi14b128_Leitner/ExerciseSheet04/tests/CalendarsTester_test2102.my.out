October 1582
Mon Tue Wed Thu Fri Sat Sun 
 16 




